packages/ui/src/composables/useAlign.ts

```diff
@@ -0,0 +1,54 @@
+import { PropType, computed } from 'vue'
```

Comment:
Can you check if this could be applied in other places and file an issue on that?